internal/statestore/redis.go

```diff
@@ -477,6 +478,15 @@ func (rb *redisBackend) FilterTickets(ctx context.Context, pool *pb.Pool, pageSi
 
 	idSet = set.Difference(idSet, idsInIgnoreLists)
 
+	// Do a little randomization to prevent dependency on return order.
```

Comment:
Could you update the PR description and the comment at here to describe the intention of this changes? e.g. `The goal of this randomization is to discourage users from taking the return order into account when implementing the MMF/Evaluator`.

Reply:
Per discussion... it already says that?